package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gossamer",
+  "version": "1.0.0",
```

Comment:
does this need to correspond to our actual current release version or no?

Reply:
No. I think this is the version to be published in the npm repo. But the tool automatically figures out the version number based on the previous release.